CMakeLists.txt

```diff
@@ -144,7 +144,11 @@ nest_process_tics_per_ms()
 nest_process_tics_per_step()
 nest_process_with_ps_array()
 nest_process_with_libltdl()
-nest_process_with_readline()
+
+if ( with-readline )
```

Comment:
This is better done in nest_process_with_readline() ... and it is already there.